taca/analysis/analysis_nanopore.py

```diff
@@ -131,7 +131,7 @@ def process_minion_qc_run(minion_run, sequencing_ongoing=False, nanoseq_ongoing=
 
                         else:
                             logger.warn('An error occurred during transfer of run {} '
-                                        'to Irma. Notifying operator.'.format(minion_run.run_id))
+                                        'to analysis cluster. Notifying operator.'.format(minion_run.run_id))
```

Comment:
if you want me to be picky: change to 
to **the** analysis cluster

Reply:
Nice catch 👍 Fixed it!